data/GombaiGorbek.md

```diff
@@ -1,5 +1,5 @@
 Ez az útvonal igazából két félből áll, melyeket Gomba és Bénye választanak el egymástól. A szakaszok akár önmagukban is megállhatják helyüket, amennyiben vissza akarnánk fordulni a településeknél - főleg az Úri felőli.
 
-A Káva és Benye közti rész tempós, enyhébb kanyarokból áll. Mivel jó minőségű, széles, fekete aszfalton haladhatunk, így akár egész komoly sebességgel lehet ezeket venni, a 60-as táblák sugallata ellenére is. A szakasz mentén gyakran földeket, mezőket láthatunk, melyeket időnként az utat szegélyző bokrok-fák takarnak el. Közepes - alacsony forgalomra számíthatunk.
+A Káva és Benye közti rész tempós, enyhébb kanyarokból áll. Mivel jó minőségű, széles, fekete aszfalton haladhatunk, így akár egész komoly sebességgel lehet ezeket venni, a 60-as táblák sugallata ellenére is. A szakasz mentén gyakran földeket, mezőket láthatunk, melyeket időnként az utat szegélyző bokrok-fák takarnak el. Főleg szürkület után a vadakra figyeljünk. Közepes - alacsony forgalomra számíthatunk.

```

Comment:
egyik oldalon erdő, másikon patak, már kétszer is láttam

Reply:
Ok, rendben van.